### ISSUE TITLE
Created Company, cant associate user to company

### ISSUE THREAD

**Issue by jkim-idtus** [grokability/snipe-it#14524]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Created the first company and confirmed it's listed in MariaDB.
However, I can't seem to associate a user to the company as none are found.

### Reproduction steps

1. create company object
2. edit any user
3. attempt to tie user to company
...

### Expected behavior

expecting to see the available list of companies to associate user with

### Screenshots

![snipeit_company_issue](https://github.com/snipe/snipe-it/assets/165394005/75e4a9f0-fe8a-4f39-a5a0-94f2cc3ceb32)

### Snipe-IT Version

v6.3.3 build 12903

### Operating System

Windows

### Web Server

IIS

### PHP Version

8.0.30

**Comment by snipe** [comment]
Please provide us with the information requested at the Getting Help documentation: 
https://snipe-it.readme.io/docs/getting-help

<blockquote><img src="https://snipe-it.readme.io/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/getting-help">Getting Help</a></strong></div><div>If you're having trouble setting up Snipe-IT, don't worry!</div></blockquote>

**Comment by jkim-idtus** [comment]
Is there really logs needed when company exists and the issue is that clicking "company" within a user results with 0 results?

**Comment by snipe** [comment]
Yes, or else I wouldn't have asked for it. There are several reasons why you'd be getting 0 results, and the first place we look is browser and app logs, to see if you have something misconfigured (which is usually the case.)

**Comment by snipe** [comment]
That dropdown does an ajax request. If the URL is set incorrectly in the .env and it cannot retrieve the results, you'll see a browser console error. If there's something wrong on the server's side (missing column because migrations weren't run, etc) that will show up in the app logs.

I'm trying to help you, but you need to give me the info I need to be able to do so - and given I've been doing this for a decade, maybe don't question why I'm asking for something.

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
